### ISSUE TITLE
Openseadragon overlay does not display image on dev server

### ISSUE THREAD

**Issue by corylown** [duke-libraries/ddr-public#124]
@seanaery I suspect this is due to the image content not loading because it's being requested over http and not https.

**Comment by corylown** [comment]
Example URL: https://repository-dev.lib.duke.edu/catalog/duke:22387

**Comment by coblej** [comment]
@corylown @seanaery For what it's worth, I can see the image when I visit the URL you referenced.  If I click on the image, I get a view that I can pan and zoom.  I'm using Safari on a Mac.  Of course, maybe something isn't working as it should anyway, since I don't really know what it should be doing.

**Comment by corylown** [comment]
It just displays a blank gray background when I click on the image in all browsers.

**Comment by seanaery** [comment]
@corylown It's an http / https issue as well as a cross-origin resource sharing issue. Just displaying the image with an <img> tag yields just a console warning about mixed content, so that still works. The Seadragon UI gets the images and data with AJAX (from a separate domain + separate protocol), so that's getting completely blocked. 

I think to fix this issue we'll need to do one or probably both of these things:
1. Enable & configure [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS) on collections-01 to permit AJAX GET requests, at least from the repository domains. I think this would still yield warnings about mixed content.
2. Ensure that our SSL certificate on collections-01 is set up in a way that makes image server data accessible at http & https URLs alike.

If we need to support both http and https image server URLs, we could use a protocol-relative IMAGE_SERVER_URL.

**Comment by corylown** [comment]
Oh right. There are some notes on BaseCamp about this:

https://basecamp.com/2517156/projects/8203819/todos/169424125#comment_283478749

**Comment by seanaery** [comment]
Bullet number 2 is now addressed. As of today, image server URLs now work at either http: or https:, e.g., https://library.duke.edu/digitalcollections/iipsrv/?IIIF=/var/www/media/derivatives/vic/ptif/viccb01001e/viccb010010010.ptif/full/!1000,1000/0/default.jpg 

For bullet 1, we decided today to try setting up a reverse proxy for AJAX requests for image server data on the repository servers. This should circumvent the origin request issue as an alternative to CORS.
